app/controllers/appointments_controller.rb

```diff
@@ -74,6 +78,6 @@ def set_appointment
 
   # Only allow a list of trusted parameters through.
   def appointment_params
-    params.require(:appointment).permit(:title, :description, :start_time, :end_time)
+    params.require(:appointment).permit(:title, :description, :start_time, :end_time, :selected_date)
```

Comment:
I left you a comment about this in the other PR.
Are you mixing up PRs?

Reply:
yes but I already reverted the changes